tests/functional/test_confirm_email_address.py

```diff
@@ -1,37 +1,31 @@
-import pytest
 from flask import url_for
-
-email_depts = [('some.one@digital.cabinet-office.gov.uk',
-                'Government Digital Service')]
+from tests.functional.user_flows import on_email_confirm_fill_in_then_continue
+from tests.functional.user_flows import yes_selected, no_selected
 
 
 class When_on_confirm_email_with_yes_selected_and_valid_email(object):
 
-    @pytest.mark.parametrize("email_address,department", email_depts)
     def it_goes_to_department_confirm_when_continue_clicked(
-            self, live_server, browser, email_address, department):
-        browser.visit(url_for('main.request_email_address', _external=True))
+            self, live_server, browser):
```

Comment:
Since all the tests in this class will use the `live_server` fixture, you can move it out of the method signature and put it in a class decorator like this:
```python
@pytest.mark.usefixtures('live_server')
class When_on_confirm_email_with_yes_selected_and_valid_email(object):
    def it_goes_to_department_confirm_when_continue_clicked(self, browser):
        ...
```